stable/hazelcast-jet/templates/tests/test-management-center.yaml

```
@@ -1,10 +1,9 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: "{{ template "hazelcast-jet-management-center.fullname" . }}-test-{{ randAlphaNum 5 | lower }}"
+  name: "{{ template "managementcenter.fullname" . }}-test-{{ randAlphaNum 5 | lower }}"
   annotations:
     "helm.sh/hook": test-success
```

```suggestion
    "helm.sh/hook": test
```
Please check https://helm.sh/docs/topics/chart_tests/